Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/StatusServiceImpl.java

```diff
@@ -127,7 +128,7 @@ public String getApplicationServer() {
                     AvailableService aService = new AvailableService();
                     aService.setServiceName(MessageFormat.format("{0} - {1}", serviceName, url.getVersion()));
                     aService.setResponseCode(PING_SERVICE.ping(url.getUrl(), !IGNORE_DEADHOST));
-
+                    aService.setServiceMessage(!aService.isAvailable()?HttpStatus.getStatusText(aService.getResponseCode()):"");
```

Comment:
@trantang  No. isAvailable() method is already doing a check against responseCode and it makes sense to use the existing method instead of re-coding the check.